proposals/lip-0071.md

```diff
@@ -70,6 +70,7 @@ We define the following constants:
 | **Configurable Constants** | | **Mainchain Value** | |
 | `BLOCK_TIME` | integer | 10 | Block time (in seconds) set in the chain configuration. |
 | `FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS` | uint32 | `1000` | It determines the percentage of rewards assigned equally to all active validators. The percentage can be obtained by dividing the value by `100`, i.e., a value of `1000` corresponds to `10%` .|
+| `DECIMAL_PERCENT_FACTOR` | uint32 | `10000` | The value is used to divide the `FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS` to obtain the minimum reward.|
```

Comment:
Do we need this?

Reply:
Yes. It is the value used in to calculate minimum reward.

Reply:
I mean, do we need a config constant? Couldn't we just use 10000 fixed? (meaning the factor would have two decimal point precision)

Reply:
Yes, we can but I would keep it to have an alignment with sdk https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/dynamic_rewards/constants.ts. Plus, it is always good to have it configurable rather than fixed constant value.

Reply:
I think there is some confusion here. 

1. In the implementation it is not a config constant, it is just a constant, fixed (you can see that is not part of defaultConfig or [moduleConfig](https://github.com/LiskHQ/lisk-sdk/blob/2d04e23b5b172a7b84cdad2669b985a51156e671/framework/src/modules/dynamic_rewards/types.ts#L20C18-L20C30)). So they just hard-coded the value 10000 to a non-config constant. Recall that we recently aligned with them on config constants, so it would be scary if we had a mis-alignment again.
2. In general, I think it would not be good to have the denominator as a configurable constant, since you then need extra checks that it should be at least the value of the numerator and potentially create errors from the user side. So it is fine I guess to just fix it to 10000, meaning percentage with double decimal precision, it's very clear.

Reply:
Updated.